packages/runtime/container-runtime/src/opLifecycle/README.md

```diff
@@ -129,6 +179,12 @@ To enable grouped batching:
     }
 ```
 
+## Note about performance and latency
+
+In terms of performance and impact on latency, Fluid does not make any guarantees or provide benchmarks as the effects of compression/chunking/grouped batching are coupled with the nature of the payloads and the configured `FlushMode`. Therefore, customers must perform the required measurements and adjust the settings according to their scenarios.
```

Comment:
I think the language is too negative. Instead of saying "no guarantees" we should say - results depends on payload size, network speed, CPU speed, payload structure (easily compressible or not).